README.md

```diff
@@ -141,92 +133,11 @@ Everytime a package/theme is installed or removed the `bundle` file is updated.
 
 ## Core Library
```

Comment:
We could remove this reference from here, it's unlikely that these functions are going to be used directly. We should add it to the architecture section.